manifests/init.pp

```diff
@@ -97,6 +97,10 @@
   Integer $queue_size                                                 = 4096,
   String $registry_file                                               = 'filebeat.yml',
 
+  Optional[String] $systemd_beat_log_opts_override                    = undef,
+  String $systemd_beat_log_opts_template                              = $filebeat::params::systemd_beat_log_opts_template,
+  String $systemd_drop_in_dir                                         = $filebeat::params::systemd_drop_in_dir,
```

Comment:
Can we rename this to `systemd_override_dir`?

Reply:
agreed
this makes a lot of sense.  not many people will under stand what a drop in directory is
I will change it to systemd_override_dir